docker-compose.yaml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+services:
+
+  dispatcher:
+    image: dispatcher
+    container_name: dispatcher
+    environment:
+      - DISP_ID=docker
+      ## Replace value with the Author IP and Port you are using:
+      - AUTHOR_IP=host.docker.internal
+      - AUTHOR_PORT=4502
+      - AUTHOR_DEFAULT_HOSTNAME=author.docker.local
+      - AUTHOR_DOCROOT=/mnt/var/www/author
+      ## Replace value with the Publisher IP and Port you are using:
+      - PUBLISH_IP=host.docker.internal
+      - PUBLISH_PORT=4503
+      - PUBLISH_DEFAULT_HOSTNAME=publish.docker.local
+      - PUBLISH_DOCROOT=/mnt/var/www/html
+      ## Replace value with the LiveCycle IP and Port you are using:
+      - LIVECYCLE_IP=127.0.0.1
+      - LIVECYCLE_PORT=8080
+      - LIVECYCLE_DEFAULT_HOSTNAME=aemforms-exampleco-dev.adobecqms.net
+      - LIVECYCLE_DOCROOT=/mnt/var/www/lc
+      - PUBLISH_FORCE_SSL=0
+      - AUTHOR_FORCE_SSL=0
+      ## Enable / Disable CRXDE access.  Production this should be disabled
+      - CRX_FILTER=deny
+      ## Allow dispatcher flush from any IP
+      - DISPATCHER_FLUSH_FROM_ANYWHERE=allow
+
+    volumes:
+      - ./ams/2.6/etc/httpd/conf:/etc/httpd/conf:ro
+      - ./ams/2.6/etc/httpd/conf.d:/etc/httpd/conf.d:ro
+      - ./ams/2.6/etc/httpd/conf.dispatcher.d:/etc/httpd/conf.dispatcher.d:ro
+      - ./ams/2.6/etc/httpd/conf.modules.d:/etc/httpd/conf.modules.d:ro
+      - ./mnt/author_docroot:/var/www/author:rw
```

Comment:
Should you maybe mention creation of these ./mnt folders in the readme or perhaps checkin those with a placeholder file?

Reply:
Good point; added both.